gobblin-core/src/main/java/gobblin/metrics/MetricNames.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (c) 2014 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.metrics;
+
+/**
+ * Contains names for all metrics generated in gobblin-core.
+ */
+public class MetricNames {
+
+  /**
+   * Extractor metrics.
+   */
+  public static class Extractor {
```

Comment:
Can you rename this class, maybe "ExtractorMetrics"? Same for the other static inner classes in this class.